TutorLizard.Web/Services/UserAuthenticationService.cs

```diff
@@ -95,9 +100,10 @@ public Task<bool> RegisterUser(string username, UserType type, string email, str
 
     public void SendActivationEmail(string userEmail, string activationCode)
     {
-        var fromAddress = new MailAddress("lizardtutoring@gmail.com", "Tutor Lizard");
+        var fromAddress = new MailAddress(_emailSettings.FromAddress, "Tutor Lizard");
         var toAddress = new MailAddress(userEmail);
-        const string fromPassword = "pvez johg nzwc enjg";
+        var fromPassword = _emailSettings.FromPassword;
+
         string subject = "Aktywacja konta";
         string body = $"Cześć tu zespół Tutor Lizard, \naby aktywować swoje konto, kliknij poniższy link: \nhttp://localhost:7092/Account/ActivateAccount?activationCode={activationCode}";
```

Comment:
Please move also the link (without the code part) to configuration. In production we would want to be able to change it to an actual domain without recompiling the whole app.